src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,33 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
```

Comment:
The `hide` method correctly sets the `hidden` property to true. However, ensure that the method's implementation matches the task description, which states that it should only hide the animal. If there are any future requirements to toggle the hidden state, a separate method should be implemented.